upstream-community-operators/twistlock/twistlock-console-operator.v0.0.9.clusterserviceversion.yaml

```diff
@@ -0,0 +1,224 @@
+  - base64data: PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9IjAgMCAxMDQuNzYgMTIwLjQyIj48ZGVmcz48c3R5bGU+LmNscy0xe2ZpbGw6IzYxMThmZjt9PC9zdHlsZT48L2RlZnM+PHRpdGxlPmNvbG9yLWxvZ29tYXJrPC90aXRsZT48ZyBpZD0iTGF5ZXJfMiIgZGF0YS1uYW1lPSJMYXllciAyIj48ZyBpZD0iTGF5ZXJfMS0yIiBkYXRhLW5hbWU9IkxheWVyIDEiPjxwb2x5Z29uIGNsYXNzPSJjbHMtMSIgcG9pbnRzPSIxMDQuNzYgNDkuNjMgMTA0Ljc2IDMxLjQgODkuMDMgNDAuNDggODkuMDMgNTguNzEgMTA0Ljc2IDQ5LjYzIi8+PHBvbHlnb24gY2xhc3M9ImNscy0xIiBwb2ludHM9Ijg5LjAzIDYyLjI1IDg5LjAzIDc5LjIxIDEwMy43MSA3MC43MyA4OS4wMyA2Mi4yNSIvPjxwb2x5Z29uIGNsYXNzPSJjbHMtMSIgcG9pbnRzPSI4Ny4wMiA0MS42NyA3MS4yMyA1MC43OSA3MS4yMyA5MS44OCA1My40NCAxMDIuMTYgNTMuNDQgMTIwLjM2IDEwNC43NiA5MC43MiAxMDQuNzYgNzIuNDQgODcuMDIgODIuNjggODcuMDIgNDEuNjciLz48cG9seWdvbiBjbGFzcz0iY2xzLTEiIHBvaW50cz0iMzUuNTEgMjcuOTggNTAuMyAxOS40MyAzNS41MSAxMC44OSAzNS41MSAyNy45OCIvPjxwb2x5Z29uIGNsYXNzPSJjbHMtMSIgcG9pbnRzPSIzNC41OSA0OS4wNSA3MC4xNyAyOC41MSA4Ny45NyAzOC43OCAxMDMuNzYgMjkuNjYgNTIuMzggMCAzNi41MiA5LjE2IDU0LjMxIDE5LjQzIDE4LjggMzkuOTQgMzQuNTkgNDkuMDUiLz48cG9seWdvbiBjbGFzcz0iY2xzLTEiIHBvaW50cz0iMTcuODcgOTkuOCAzMi41OSA5MS4zIDE3Ljg3IDgyLjgxIDE3Ljg3IDk5LjgiLz48cG9seWdvbiBjbGFzcz0iY2xzLTEiIHBvaW50cz0iNTEuMzggODEuNjEgMTUuNzkgNjEuMDYgMTUuNzkgNDAuNTIgMTQuNzkgMzkuOTQgMTQuNzkgMzkuOTQgMCAzMS40IDAgOTAuNzIgMTUuODcgOTkuODkgMTUuODcgNzkuMzQgNTEuMzggOTkuODQgNTEuMzggODEuNjEiLz48cG9seWdvbiBjbGFzcz0iY2xzLTEiIHBvaW50cz0iNTEuNDQgMTAyLjE5IDM1LjU5IDkzLjA0IDM1LjU5IDExMS4yNyA1MS40NCAxMjAuNDIgNTEuNDQgMTAyLjE5Ii8+PHBvbHlnb24gY2xhc3M9ImNscy0xIiBwb2ludHM9IjE3LjggNTkuOSAzMy41OCA2OS4wMiAzMy41OCA1MC43OSAxNy44IDQxLjY3IDE3LjggNTkuOSIvPjxwb2x5Z29uIGNsYXNzPSJjbHMtMSIgcG9pbnRzPSIxIDI5LjY2IDE2LjggMzguNzggMzIuNjYgMjkuNjIgMTYuODcgMjAuNSAxIDI5LjY2Ii8+PHBvbHlnb24gY2xhc3M9ImNscy0xIiBwb2ludHM9IjY4LjIzIDcwLjczIDUyLjQ5IDYxLjY0IDM2LjY1IDcwLjc5IDUyLjM4IDc5Ljg3IDY4LjIzIDcwLjczIi8+PHBvbHlnb24gY2xhc3M9ImNscy0xIiBwb2ludHM9IjUxLjM4IDU5Ljk3IDUxLjM4IDQxLjY3IDM1LjU5IDUwLjc5IDM1LjU5IDY5LjA5IDUxLjM4IDU5Ljk3Ii8+PHBvbHlnb24gY2xhc3M9ImNscy0xIiBwb2ludHM9Ijg2IDM5Ljk1IDcwLjE3IDMwLjgyIDU0LjQxIDM5LjkyIDcwLjIzIDQ5LjA1IDg2IDM5Ljk1Ii8+PHBvbHlnb24gY2xhc3M9ImNscy0xIiBwb2ludHM9IjUzLjM4IDU5Ljg0IDY5LjIzIDY4Ljk5IDY5LjIzIDUwLjc5IDUzLjM4IDQxLjY0IDUzLjM4IDU5Ljg0Ii8+PHBvbHlnb24gY2xhc3M9ImNscy0xIiBwb2ludHM9IjUzLjM4IDgxLjYxIDUzLjM4IDk5Ljg4IDY5LjIzIDkwLjczIDY5LjIzIDcyLjQ2IDUzLjM4IDgxLjYxIi8+PC9nPjwvZz48L3N2Zz4=
+    mediatype: image/svg+xml
+  displayName: Twistlock Console Operator
+  install:
+    spec:
+      deployments:
+      - name: twistlock-console-helm-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: twistlock-console-helm-operator
+          strategy: {}
+          template:
+            metadata:
+              labels:
+                name: twistlock-console-helm-operator
+            spec:
+              containers:
+              - env:
+                - name: WATCH_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.namespace
```

Comment:
Please change this to `metadata.annotations['olm.targetNamespaces']`

Reply:
will do. resubmitting with fixes.